Mathlib/Algebra/Homology/HomotopyCategory/Pretriangulated.lean

```diff
@@ -9,8 +9,13 @@ import Mathlib.CategoryTheory.Triangulated.Functor
 
 /-! The pretriangulated structure on the homotopy category of complexes
 
+<<<<<<< HEAD
```

Comment:
Seems like something went wrong with merge conflict resolution.

Reply:
Oops. I have just fixed this.